### DISCUSSION TITLE
CCU3 lässt sich nicht auf 3.75.6 updaten

### DISCUSSION THREAD

**Discussion by MbdL-priv** [OpenCCU/OpenCCU#2704] [🇩🇪 Fragen&Antworten]
hallo,

obwohl mir die CCU3 die aktuelle Firmware anzeigt, kann ich das Update nicht durchführen. habe schon versucht über "automatisches Update", recovery Modus, manuelles Update. Interessanterweise wird das Update im Addon System Update/rmupdate nicht angezeigt.

**Comment by Baxxy13** [comment]
Geht es um CCU3 original Firmware oder wirklich um RaspberryMatic die auf der CCU3 Hardware läuft?
Im ersten Fall wärst du hier falsch, aber gehen wir mal vom 2. Fall aus...

> kann ich das Update nicht durchführen

Sehr unpräzise Beschreibung. Was passiert denn genau?

Aus dem Bauch heraus:
Du hast nicht genug freien Speicher weil du beim Umstieg von CCU3 zu RaspberryMatic den Werksreset nicht gemacht hast.
Screenshot der Hilfeseite könnte hier Klarheit bringen.

Das erwähnte AddOn ist mehr für Probleme als für Nutzen bekannt, ich würde versuchen es loszuwerden.

**Reply by MbdL-priv** [reply]
Danke, ja es geht um rasberrymatic, was ich schon seit viele Monaten auf der CCU3 drauf habe - deswegen die Verwunderung, dass das Update nicht mehr geht. Das komische ist, dass alles „durchläuft“ und nach dem Neustart ist immer noch die alte Firmware drauf. Werksreset heute noch mal gemacht, Speicher habe ich glaube ich genug. Screenshot der hilfeseite anbei. Tausend Dank! 
![IMG_8386](https://github.com/jens-maus/RaspberryMatic/assets/165407605/1385c2fb-6501-4109-9dde-b9197d27f4e2)
.

**Reply by Baxxy13** [reply]
Ok, sieht jetzt für mich soweit gut aus.
Du hättest versuchen können nach dem Werksreset zu updaten und danach dein Backup zu restoren.

Ich würde die aktuelle RaspberryMatic auf eine "neue" SD Karte flashen, booten und dort das Backup einspielen.
Warum das Update nicht sauber durchläuft kann ich leider nicht sagen.
